scripts/start-workers.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# EPFL Bigdata Course 2015, remote actor starter/stopper script V.1
+
+# Start and stop the remote actor system on our worker nodes
+
+if [ $# -ne "1" ]; then
+	echo "Usage: ./start-workers.sh <start | stop | status>"
+	exit
+fi
+
+DOMAINNAME=".cloudapp.net"
+HOSTS=(ts-1-021qv44y ts-2) #ts-3 ts-4 ts-5 ts-6 ts-7 ts-8)
+USERS=(merlin dennis) #ts3 jakub ts-5 jakob ts-7 ts-8)
+PORTS=(22 22) # 22 22 22 22 65530 52640)
+
+
+for i in "${!HOSTS[@]}"; do
+	HOST=${HOSTS[$i]}
+	USER=${USERS[$i]}
+	PORT=${PORTS[$i]}
+	echo "Connecting to $USER@$HOST$DOMAINNAME port $PORT"
+
+	case "$1" in
+		start)
+			# The following long line does:
```

Comment:
Nice doc :)
Wouldn't it be possible to split it to several lines using `\`? (I'm not too familiar with the syntax)